core/src/components/picker-column/picker-column.tsx

```diff
@@ -159,7 +185,7 @@ export class PickerColumnCmp implements ComponentInterface {
       }
 
       // Update transition duration
-      if (this.noAnimate) {
+      if (this.noAnimate || !this.shouldAnimteOnRender) {
```

Comment:
Why is this needed?

Reply:
If a user ends up adding an option and has the animation on, then the option will render on top of the first option then slide down to the bottom. With animation turned off, it will automatically render at the bottom.